### PR TITLE
allow for file_format to be nil

### PR DESCRIPTION
**Pull request by dnoneill**

### CODE REVIEW

File: app/views/layouts/catalog_result.html.erb [hunk 1]
```diff
@@ -8,7 +8,7 @@
   </section>
 
   <section class="col-lg-5">
-    <% if (@document.file_format.include?('JPEG') || @document.item_viewer.iiif) && !@document[Settings.FIELDS[:GEOREFERENCED]] %>
+    <% if (@document.file_format&.include?('JPEG') || @document.item_viewer.iiif) && !@document[Settings.FIELDS[:GEOREFERENCED]] %>
```
Comment (jcoyne) [inline app/views/layouts/catalog_result.html.erb:11]: Can you make this a `jpeg?` method in SolrDocument?

**Review by jcoyne — CHANGES_REQUESTED** [review]

File: app/models/solr_document.rb [hunk 2]
```diff
@@ -50,4 +50,10 @@ def searchworks_url
   # and Blacklight::Solr::Document#to_semantic_values
   # Recommendation: Use field names from Dublin Core
   use_extension(Blacklight::Document::DublinCore)
+
+  # Check to see if view is an image
+  # Used for georeferencing message
+  def is_image?
+    file_format&.include?('JPEG')
```
Comment (jcoyne) [inline app/models/solr_document.rb:57]: Would you please add a test for this method, including a case when file_format is nil?

**Review by jcoyne — CHANGES_REQUESTED** [review]

**Review by jcoyne — APPROVED** [review]
Thanks!
